README.md

```diff
@@ -2,6 +2,9 @@ GitHub Action for refreshing dummy self hosted runner.
 
 - Create or refresh dummy runner for target organization.
 
+**Table of Contents**
+
+[TOC]
```

Comment:
GitHub Flavored Markdown doesn't have syntax for Table of Contents. This line is WIP?